### ISSUE TITLE
[BUG] Plex 'get' Request Failed

### ISSUE THREAD

**Issue by korumera** [DEFENDORe/pseudotv#14]
When I check the PseudoTV.exe window it shows the following error:

(node:3288) UnhandledPromiseRejectionWarning: Plex 'Get' request failed. URL: https://xxxx:32400/livetv/dvrs
(node:3288) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)

And it happens every time after it tries to update the XMLTV. I use the URL instead of the file because I cant add the file it self Plex won't accept it.

**Comment by powdermilkman** [comment]
Are plex and pseudotv running on the same machine?

**Comment by korumera** [comment]
No, pseudo is running on my windows 10 machine and plex is running on my synology nas (not in a docker container). Ive posted on the plex thread that on the local path wont work because it's not on the same machine.

**Comment by thepcnerdonline** [comment]
Wouldn't the XML be on the machine that Pseduo is running on though ... it would be local to the win 10 mcahine if I am understanding things correctly?

**Comment by korumera** [comment]
The XML is on the local machine yes, but you have plex ain't so if you enter the path X:\.pseudotv\xmltv.xml that won't work, because plex can't reach that path.

**Comment by powdermilkman** [comment]
Yeah, we'll have to see if we can fix updating by url as that is a completely valid use case.

**Comment by korumera** [comment]
This issue will be solved if the direct.plex url is used when setting up the plex server in PseudoTV. 
It's https://<realIP>.<token>.plex.direct:22686. I'm using the fork that uses plex for transconding and set it up like this and the error is gone.
